.github/workflows/ar-ci.yml

```diff
@@ -0,0 +1,73 @@
+name: "AR CI"
+
+on:
+  push:
+    paths-ignore:
+      - "dotcom-rendering/**"
+
+  workflow_dispatch:
+
+# Allow queued workflows to interrupt previous runs
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.ref }}
+  cancel-in-progress: true
+
+# Allow GitHub to request an OIDC JWT ID token, to use with aws-actions/configure-aws-credentials
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    # Checkout the repository
+    steps:
+      - uses: actions/checkout@v4
+
+      # Get the desired version of Node installed
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: .nvmrc
+
+      - name: Install dependencies
+        run: |
+          npm i -g yarn@1.x
+          yarn --silent --frozen-lockfile
+
+      # Execute some tasks from the `apps-rendering/package.json` file, using `yarn`
+      - name: Build and package
+        working-directory: apps-rendering
+        run: |
+          yarn test
+          yarn build:client:prod
+          yarn build:server:prod
+          yarn copy-manifest
+          yarn copy-fonts
+          yarn synth
+          zip -r dist/server/mobile-apps-rendering.zip dist/server
```

Comment:
We are replacing the deprecated [`node-riffraff-artifact`](https://github.com/guardian/node-riffraff-artifact) with [`actions-riff-raff`](https://github.com/guardian/actions-riff-raff). `node-riffraff-artifact` was zipping under the hood the contents of `dist/server` and producing `mobile-apps-rendering.zip`. We can verify this in its [config](https://github.com/guardian/dotcom-rendering/blob/main/apps-rendering/artifact.json#L6-L8) file. We are now adding a command to do the same.